app/decorators/course_decorator.rb

```diff
@@ -75,7 +75,7 @@ def scholarship_amount
   end
 
   def salaried?
-    object.funding_type == "salary"
+    object.funding_type == "salary" || object.funding_type == "apprenticeship"
```

Comment:
https://github.com/DFE-Digital/teacher-training-api/blob/2fcb258955cab533f2845c67d3e9d85bf1c4546e/app/models/course.rb#L311

Should it not be using this instead?

Reply:
Possibly but I copied what search-and-compare-ui does which is the same as the preview in publish. Possible refactor post-launch for both preview and show